packages/rolldown/src/binding.d.ts

```diff
@@ -745,12 +745,6 @@ export interface EcmaScriptModule {
   importMetas: Array<Span>
 }
 
-export interface ErrorLabel {
```

Comment:
These types seem deleted unexpected.

Reply:
Not sure how I got into this. I just ran `just build`.

I guess this is due to the crate `oxc_transformer_binding` we rely on is updated but we didn't update the binding in the same time.